acceptance-tests/tests/src/test/resources/clique/clique.json

```diff
@@ -1,12 +1,7 @@
 {
   "config": {
     "chainId": 4,
-    "homesteadBlock": 1,
```

Comment:
These seem orthogonal to the PR scope, which is already quite large due to the AT changes.  Please move to a different PR.  Same with ibft and qbft.

Reply:
I had to do this to make acceptance tests pass or the first block cannot be minted.

Reply:
I don't follow.  Acceptance tests were working before, right?  What changed to require this change?  The protocol specs should simply upgrade each block as it passes the milestone, why does it suddenly need to start at a future fork for reference tests?

Reply:
It looks like I tried to send a eip155 transaction to a frontier block, and it didn't really like that.

Reply:
My thought then is that the ATs for eip-155 transactions should wait until a certain block is mined or a separate set of AT genesis files should be used that start right after spurious monkey.

Reply:
Not to drag this thread along too much, but there is a fine point here that this change introduces, which pretty much says you'd better start your chain after a certain ethereum hard fork.

I'm not quite sure what the point of having private chains with frontier blocks is.

It would also make sense if Besu could deal with EIP-155 transactions and wait out until the hard fork has happened to introduce them. I guess the same thing will happen with EIP-1559 type transactions next, if/when it gets adopted.

Reply:
We still have to be able to full import mainnet and classic, even if unprotected transactions are prohibited in a future fork.  So I am still uncomfortable with the broad changes occurring to the ATs implied by these changes.  I am of the opinion a different solution should be found, such as adding `clique_berlin.json` et.al, or ensuring that the ATs are past the spurious dragon blocks before submitting EIP-155 transactions.  I am also uneasy moving wholesale to EIP-155 transactions in the ATs.  While they should be present the should not make pre-EIP-155 TX formats extinct in the AT stack.

Reply:
So you’re saying:
- tie eip-155 tx flag to the protocol schedule
- revert all genesis file changes
- Make all tests pass again

I’ll do it.

Reply:
Done